specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimapis.json

```diff
@@ -0,0 +1,1878 @@
+        "x-ms-examples": {
+          "ApiManagementPutApis": {
+            "$ref": "./examples/ApiManagementPutApis.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiCreateOrUpdateParameter"
+            },
+            "description": "Create or update parameters."
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "ETag of the Api Entity. For Create Api Etag should not be specified. For Update Etag should match the existing Entity or it can be * for unconditional update."
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "API was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/ApiContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
+            "description": "API was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ApiContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Apis"
+        ],
+        "operationId": "Api_Update",
+        "description": "Updates the specified API of the API Management service instance.",
+        "x-ms-examples": {
+          "ApiManagementPatchApis": {
+            "$ref": "./examples/ApiManagementPatchApis.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiUpdateContract"
+            },
+            "description": "API Update Contract parameters."
+          },
+          {
+            "name": "If-Match",
```

Comment:
>"If-Match" [](start = 20, length = 10)

Why not add it to the parameters section? This way you can avoid lots of duplication :)

Reply:
agree, but description is different. 

---
In reply to: [163342706](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163342706) [](ancestors = 163342706)